src/custom_modules/mail.js

```diff
@@ -563,6 +563,50 @@ async function sendTaxDeductions(taxDeductionRecord, year) {
   }
 }
 
+/** 
+ * @param {string} KID
+ * @param {"STOPPED" | "AMOUNT" | "CHARGEDAY" | "SHARES"} change What change was done
```

Comment:
Is this correct? I see that CANCELLED is used further down in the code f.ex.
I think the states are different for avtalegiro than for vipps